src/types/ScholarshipAmount.ts

```diff
@@ -132,13 +132,13 @@ namespace ScholarshipAmount {
       case AmountType.FullTuition:
         return 'Full Tuition';
       case AmountType.Fixed:
-        return `$${amount.min}`;
+        return `${amount.min}`;
       case AmountType.Varies:
         if (!amount.min && !amount.max) return 'Varies';
         if (amount.min && amount.max && amount.max !== RANGE_MAX) {
-          return `$${amount.min}-$${amount.max}`;
+          return `${amount.min}-${amount.max}`;
         }
-        return amount.min ? `$${amount.min}+` : `Up to $${amount.max}`;
+        return amount.min ? `${amount.min}+` : `Up to ${amount.max}`;
```

Comment:
Hmm now it'll say `$Up to 1000`. Maybe we should change the wording? E.g. `1000 max` or something else